MoonVision/moonbox-docker
# Feature/mn 366 upgrade torch

# MN-366 Upgrade Torch

## How to test

On a GPU machine with `cuda >= 11.0`: 

```shell
docker pull moonvision/dev-moonbox:cuda-latest
docker run -it --rm --gpus all moonvision/dev-moonbox:cuda-latest python -c "\
import torch, torchvision;\
assert torch.cuda.is_available();\
print(torch.__version__, torchvision.__version__)
"
```
Expect: `1.11.0+cu113 0.12.0+cu113`.
